tests/run-test.sh.in

```diff
@@ -38,7 +38,12 @@ GNOME_SHELL_TESTSDIR="$srcdir/"
 GNOME_SHELL_JS="$srcdir/../js"
 GNOME_SHELL_DATADIR="$builddir/../data"
 
-export GI_TYPELIB_PATH GJS_PATH GJS_DEBUG_OUTPUT GJS_DEBUG_TOPICS GNOME_SHELL_TESTSDIR GNOME_SHELL_JS GNOME_SHELL_DATADIR LD_PRELOAD
+# This bit is needed not to break the tests because of libeos-shell-fx.so, since
+# that private library is not directly linked from the tests and thus it will be
+# dinamically loaded at run-time, causing the tests to fail if this is not defined.
```

Comment:
No worries if this is a can of worms, but couldn't we instead link the new library directly from the tests instead of this, since that's what the comment suggests the problem is?